components/match2/wikis/smite/match_summary.lua

```diff
@@ -0,0 +1,275 @@
+---
+-- @Liquipedia
+-- wiki=smite
+-- page=Module:MatchSummary
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local CustomMatchSummary = {}
+
+local Abbreviation = require('Module:Abbreviation')
+local Array = require('Module:Array')
+local CharacterIcon = require('Module:CharacterIcon')
+local Class = require('Module:Class')
+local DateExt = require('Module:Date/Ext')
+local Icon = require('Module:Icon')
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+
+local DisplayHelper = Lua.import('Module:MatchGroup/Display/Helper')
+local MatchSummary = Lua.import('Module:MatchSummary/Base')
+local Opponent = Lua.import('Module:Opponent')
+
+local MAX_NUM_BANS = 7
```

Comment:
Is this number correct? I googled a bit and it seemed to be 3?

Reply:
Will update to be 5 (they were previously 3 and moved up to 5 currently)